lib/torba/verify.rb

```diff
@@ -1,7 +1,14 @@
 require "torba"
 
-if STDOUT.tty?
-  Torba.pretty_errors { Torba.verify }
-else
-  Torba.verify
+# We purposely skip verification during Rake execution, because Rake is used to
+# run the Torba:RakeTask during the "assets:precompile" stage of deployment. In
+# this case it is OK that the Torbafile hasn't been packed yet, because we are
+# about to do so.
+#
+unless defined?(Rake::Task)
```

Comment:
BTW, if we treat this as a general approach, then `Rake` constant could also be not defined.